### PR TITLE
Pin Cassandra version because ARM build is missing in 3.11.10+

### PR DESCRIPTION
**Pull request by sw-yx**
## Description, Motivation and Context

ran into this error while doing it on my machine, adding a note to the docs to be helpful to others

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ tctl --ns test-namespace namespace register -rd 1
 
 Get started building Workflows with a [Go sample](https://github.com/temporalio/samples-go), [Java sample](https://github.com/temporalio/samples-java), or write your own using one of the [SDKs](https://docs.temporal.io/docs/sdks-introduction).
 
+> Note for running this `docker-compose` on M1 Apple Silicon: you may run into a `no matching manifest for linux/arm64/v8 in the manifest list entries` error. To resolve this, [specify the platform](https://stackoverflow.com/questions/65456814/docker-apple-silicon-m1-preview-mysql-no-matching-manifest-for-linux-arm64-v8) for the Cassandra container.
```
Comment (alexshtin) [inline README.md:43]: Why not just to add:
```
platform: linux/x86_64
```
to `docker-compose*.yaml` files? There should be no harm doing it.

Comment (sw-yx) [reply README.md:43]: actually as I found out we do need the ARM build, so i need to change this run note altogether...

File: README.md [hunk 2]
```diff
@@ -40,6 +40,8 @@ tctl --ns test-namespace namespace register -rd 1
 
 Get started building Workflows with a [Go sample](https://github.com/temporalio/samples-go), [Java sample](https://github.com/temporalio/samples-java), or write your own using one of the [SDKs](https://docs.temporal.io/docs/sdks-introduction).
 
+> Note for running this `docker-compose` on M1 Apple Silicon: you may run into a `no matching manifest for linux/arm64/v8 in the manifest list entries` error. This is a temporary outcome of the Cassandra Docker image [not publishing ARM builds](https://github.com/docker-library/cassandra/issues/222). For the time being, pin the version to Cassandra `v3.11.9` instead of just `3.11`.
+
```
Comment (sw-yx) [inline README.md:44]:
```suggestion
```

**Review by alexshtin — APPROVED** [review]

**Comment by sw-yx** [comment]
technically we can undo this already bc cassandra image arm builds are being published again.. but 4.0 is not quite ready yet: https://github.com/docker-library/cassandra/issues/222#issuecomment-827839314

**Pull request merged**